client/web/src/components/panels/Document.vue

```diff
@@ -328,25 +426,39 @@ export default defineComponent({
 			select_tool(toolName);
 		},
 		async viewModeChanged(toolIndex: number) {
-			console.log(`The view mode has been changed to index match the icon at index ${toolIndex}`);
-		},
+			console.log(
+				`The view mode has been changed to index match the icon at index ${toolIndex}`
+			);
+		}
 	},
 	mounted() {
-		registerResponseHandler(ResponseType.UpdateCanvas, (responseData) => {
-			this.viewportSvg = responseData.split("\n").map((shape, i) => shape.replace("#fff", `#${Math.floor(Math.abs(Math.sin(i + 1)) * 16777215).toString(16)}`)).join("\n");
+		registerResponseHandler(ResponseType.UpdateCanvas, responseData => {
+			this.viewportSvg = responseData
+				.split("\n")
+				.map((shape, i) =>
+					shape.replace(
+						"#fff",
+						`#${Math.floor(
+							Math.abs(Math.sin(i + 1)) * 16777215
+						).toString(16)}`
+					)
+				)
+				.join("\n");
 		});
-		registerResponseHandler(ResponseType.SetActiveTool, (responseData) => {
+		registerResponseHandler(ResponseType.SetActiveTool, responseData => {
 			this.activeTool = responseData;
 		});
 
 		window.addEventListener("keyup", (e: KeyboardEvent) => this.keyUp(e));
-		window.addEventListener("keydown", (e: KeyboardEvent) => this.keyDown(e));
+		window.addEventListener("keydown", (e: KeyboardEvent) =>
+			this.keyDown(e)
+		);
 	},
 	data() {
 		return {
 			viewportSvg: "",
-			activeTool: "Select",
+			activeTool: "Select"
```

Comment:
Please add back the closing comma here and everywhere.

Reply:
Vetur formatter..